server/schedule/operator_controller.go

```diff
@@ -548,3 +583,65 @@ func (o *OperatorRecords) Put(op *Operator, status pdpb.OperatorStatus) {
 	}
 	o.ttl.Put(id, record)
 }
+
+// exceedStoreLimit returns true if the store exceeds the cost limit after adding the operator. Otherwise, returns false.
+func (oc *OperatorController) exceedStoreLimit(ops ...*Operator) bool {
+	opInfluence := NewTotalOpInfluence(ops, oc.cluster)
+	for storeID := range opInfluence.storesInfluence {
+		if oc.storesLimit[storeID] == nil {
+			rate := oc.cluster.GetStoreBalanceRate()
+			oc.newStoreLimit(storeID, rate)
+		}
+		stepCost := opInfluence.GetStoreInfluence(storeID).StepCost
+		if stepCost == 0 {
+			continue
+		}
+		available := oc.storesLimit[storeID].Available()
+		storeLimit.WithLabelValues(strconv.FormatUint(storeID, 10), "available").Set(float64(available) / float64(RegionInfluence))
```

Comment:
I think there is no necessary to use `RegionInfluence` as 1000 here, and it is really confused.

The idea of `ratelimit` is issuing tokens. The `Available()` method returns the number of remaining tokens which should be an integer. It is meaningless to say 0.5 tokens.

I think the code here cannot work as expected. If the metric shows there are 0.5 tokens available, there is nothing available in fact, this is because of that the step cost of one operation if 1000 which exceeds 0.5*1000.